partner_statement/report/report_statement_common.py

```diff
@@ -44,15 +44,17 @@ def _show_buckets_sql_q1(self, partners, date_end, account_type):
                 ELSE l.balance + sum(coalesce(pc.amount, 0.0))
             END AS open_due,
             CASE WHEN l.balance > 0.0
-                THEN l.amount_currency - sum(coalesce(pd.amount_currency, 0.0))
-                ELSE l.amount_currency + sum(coalesce(pc.amount_currency, 0.0))
+                THEN l.amount_currency - sum(coalesce(pd.debit_amount_currency, 0.0))
```

Comment:
@MiquelRForgeFlow Your advice here. The apr record has changed, but I don't have any v14.0 forex records to check what the right thing to do is, but my gut tells me this is wrong.

Reply:
Ok, I will check.

Reply:
@MiquelRForgeFlow when you have a chance :) or anyone else interested.